README.md

```diff
@@ -4,8 +4,8 @@ Neptun PowerUp!
 Felturbózza a Neptun-odat. Gyorsabb tárgy- és vizsgafelvétel, kidobás elleni védelem,
 automatikus belépés, könnyebben kezelhető felület, tárolható belépési adatok, és még sok más.
 
-**Hivatalos weboldal:**<br>
```

Comment:
Az `npu.herokuapp.com` és a `secure.solymosi.eu/blog/neptun-powerup` linkek ugyanarra mutatnak. Ezenfelül a kódban is az `npu.herokuapp.com` van használva, pl. a statisztika modulban.